tests/validation/configs/README.md

```diff
@@ -0,0 +1,126 @@
+# Test Configuration
+
+This directory contains configuration files for the Media Transport Library validation test suite. These files define the test environment, network topology, and test parameters.
+
+## Configuration Files
+
+### test_config.yaml
+
+This file contains general test environment settings:
+
+```yaml
+build: /path/to/mtl/build
+mtl_path: /path/to/mtl
+media_path: /mnt/media
+capture_cfg:
+  enable: false
+  test_name: test_name
+  pcap_dir: /mnt/ramdisk/pcap
+  capture_time: 5
+  interface: null
+ramdisk:
+  media: 
+    mountpoint: /mnt/ramdisk/media
+    size_gib: 32
+  pcap:
+    mountpoint: /mnt/ramdisk/pcap
+    size_gib: 768
+```
+
+#### Key Parameters
+
+- **build**: Path to the Media Transport Library build directory
+- **mtl_path**: Path to the Media Transport Library installation
+- **media_path**: Path to the directory containing test media files
+- **capture_cfg**: Network packet capture configuration
+  - **enable**: Enable/disable packet capture
+  - **test_name**: Name prefix for capture files
+  - **pcap_dir**: Directory to store capture files
+  - **capture_time**: Duration of packet capture in seconds
+  - **interface**: Network interface to capture from
+- **ramdisk**: RAM disk configuration for high-performance testing
+  - **media.mountpoint**: Mount point for media RAM disk
+  - **media.size_gib**: Size of media RAM disk in GiB
+  - **pcap.mountpoint**: Mount point for packet capture RAM disk
+  - **pcap.size_gib**: Size of packet capture RAM disk in GiB
+
+### topology_config.yaml
```

Comment:
please add path to the topology_config.yaml 
(markdown supports interactive paths wchich would be perfect)